src/main/java/com/bynder/sdk/model/Settings.java

```diff
@@ -44,6 +49,12 @@ public Settings(final URL baseUrl, final String consumerKey, final String consum
         this.tokenSecret = tokenSecret;
```

Comment:
shouldn't we add here: this.httpConnectionSettings = new HttpConnectionSettings();